config/global.php

```diff
@@ -31,8 +31,8 @@
 
     'path.cache' => DI\string('{path.tmp}/cache/tracker/'),
 
-    'Piwik\Cache\Eager' => function (ContainerInterface $c) {
-        $backend = $c->get('Piwik\Cache\Backend');
+    'Matomo\Cache\Eager' => function (ContainerInterface $c) {
```

Comment:
@diosmosis  is this change name change needed from `Piwik\Cache\Eager' to `Matomo\Cache\Eager? Doesn't it break API? In reality, it is still a `Piwik\Cache\Eager` or not?

Reply:
Oh I see, yes that would be a BC break. I'll keep it and make it an alias to `Matomo\Cache\Eager` (we need both for autowiring).

Reply:
Actually, that might break since the types aren't the same, I'll have to think on this some.